uint8 id
uint8 cmd_mode
uint16 cmd_data
uint16 address
uint16 data_type
float32 value